lib/rules/shorthand-property-no-redundant-values/index.cjs

```diff
@@ -140,6 +140,16 @@ const rule = (primary, _secondaryOptions, context) => {
 					break;
 				}
 
+				// `var()` should be ignored. E.g.
+				//
+				//   --padding: 20px 5px;
+				//   padding: 0 var(--padding) 0;
+				//
+				if (typeGuards.isValueFunction(valueNode) && valueNode.value.toLowerCase() === 'var') {
+					shouldIgnore = true;
```

Comment:
suggestion
refactor

What about a `return` inside both this `if` and the preceding one?
i.e. `shouldIgnore` flag seems unnecessary